lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -583,30 +583,97 @@ def test_get_html_with_mocked_edx_video_id(self):
         )
 
     def test_get_html_with_existing_edx_video_id(self):
-        # create test profiles and their encodings
+        """
+        Tests the `VideoModule` `get_html` where `edx_video_id` is given and related video is found
```

Comment:
`VideoModule` is a module name. According to recommendations stated [here](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html), this is how we are suppose to write.
Thanks